backend/utils/graph_db_utils_test.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import ANY, MagicMock
+from neo4j import Driver, Session
+import pytest
+import utils.graph_db_utils
+
+def test_database_connectivity_is_healthy(mocker):
+    mock_driver = mocker.patch("utils.graph_db_utils.driver", return_value=MagicMock(spec=Driver))
```

Comment:
I've had to fully-qualify the calls to the methods being tested.

Initially, I tried the import below...
```
from utils.graph_db_utils import test_connection
```
... but this resulted in the `PyTest` warning:
```
Expected None, but utils/graph_db_utils_test.py::test_connection returned False,
which will be an error in a future version of pytest
```
... also this resulted in the mock driver being ignored and the test trying to make real HTTP calls to the neo4j database.